stylesheets/README.md

```diff
@@ -0,0 +1,6 @@
+# New File
+
+When adding a new stylesheet to the repository, currently, after merge, it must be added to https://liquipedia.net/commons/Special:ResourceLoaderArticles
+# Post Merge
+
+After merge to main, currenctly, the `Resourceloaderarticles-cacheversion` needs to be incremented. It can be found at https://liquipedia.net/commons/Special:LiquipediaMediaWikiMessages/edit/22
```

Comment:
```suggestion
After merge to main, currently, the `Resourceloaderarticles-cacheversion` needs to be incremented. It can be found at https://liquipedia.net/commons/Special:LiquipediaMediaWikiMessages/edit/22
```